middleware.go

```diff
@@ -16,6 +16,8 @@ import (
 
 const (
 	tracerName = "github.com/riandyrn/otelchi"
+
+	traceResponseHeaderKey = "X-Trace-ID"
```

Comment:
I think we should rename `traceResponseHeaderKey` to `defaultTraceResponseHeaderKey`.

This will make it easy for the reader to understand that this is the default header key.